tests/migrations/utils.py

```diff
@@ -149,3 +254,48 @@ def add_reply(session: Session, journalist_id: int, source_id: int) -> None:
     )
     """
     session.execute(text(sql), params)
+
+
+def mark_file_as_seen(session: Session, file_id: int, journalist_id: int) -> None:
+    params = {
+        "file_id": file_id,
+        "journalist_id": journalist_id,
+    }
+    sql = """
+    INSERT INTO seen_files (file_id, journalist_id)
+    VALUES (:file_id, :journalist_id)
+    """
+    try:
+        session.execute(text(sql), params)
+    except IntegrityError:
+        pass
```

Comment:
Under which situations would an IntegrityError be raised here and for `mark_message_as_seen` and `mark_reply_as_seen` ? Curious as to why we are `pass`ing here

Reply:
good catch, this is pointless (might have made sense from an earlier iteration and accidentally left it in). there shouldn't be an integrity error here.